plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/AnnotationManagerImpl.kt

```diff
@@ -361,7 +361,7 @@ abstract class AnnotationManagerImpl<G : Geometry, T : Annotation<G>, S : Annota
         annotation
       }
 
-      geoJsonSource.featureCollection(FeatureCollection.fromFeatures(features))
+      geoJsonSource.featureCollection(FeatureCollection.fromFeatures(features), onDataParsed = {})
```

Comment:
can `onDataParsed` be made optional so we don't need to provide an empty implementation?

Reply:
It is optional and `null` is used as default providing sync implementation.
Reason for using empty callback here is to force using async mode that actually uses one worker thread under the hood and message queue with `post`. 
That actually means that loading initial style + updating here is synchronised on worker thread.